content/intro-to-storybook/vue/es/using-addons.md

```diff
@@ -0,0 +1,236 @@
+
+## Usando Knobs para encontrar casos de borde
+
+Además, con un fácil acceso para editar los datos pasados ​​a un componente, los QA Engineers o los UI Engineers ahora pueden llevar un componente al límite. Como ejemplo, ¿qué le sucede a `Task` si nuestro elemento de la lista tiene un string _GIGANTESCO_?
+
+![¡Oh no! ¡El contenido de la extrema derecha está cortado!](/intro-to-storybook/addon-knobs-demo-edge-case.png) 😥
+
+¡Gracias a poder probar rápidamente diferentes entradas a un componente, podemos encontrar y solucionar estos problemas con relativa facilidad! Arreglemos el problema de desbordamiento agregando un estilo a `Task.vue`:
+
+```html
+<!--src/components/Task.vue>-->
+
+<!-- This is the input for our task title.
+     In practice we would probably update the styles for this element but for this tutorial,
+     let's fix the problem with an inline style:-->
+<input
+  type="text"
+  :readonly="true"
+  :value="this.task.title"
+  placeholder="Input title"
+  style="text-overflow: ellipsis;"
+/>
+```
+
+![Mucho mejor.](/intro-to-storybook/addon-knobs-demo-edge-case-resolved.png) 👍
+
+## Agregar una nueva historia para evitar regresiones
+
+Por supuesto, siempre podemos reproducir este problema ingresando la misma entrada en los mandos, pero es mejor escribir una historia fija para esta entrada. Esto aumentará sus pruebas de regresión y describirá claramente los límites de los componentes para el resto de su equipo.
+
+Agreguemos una historia para el caso de texto largo en `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+
+const longTitle = `This task's name is absurdly large. In fact, I think if I keep going I might end up with content overflow. What will happen? The star that represents a pinned task could have text overlapping. The text could cut-off abruptly when it reaches the star. I hope not!`;
+
+// default task state
+export const Default = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: object('task', { ...taskData }),
+    },
+  },
+  methods: actionsData,
+});
+// pinned task state
+export const Pinned = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_PINNED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+// archived task state
+export const Archived = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        state: 'TASK_ARCHIVED',
+      },
+    },
+  },
+  methods: actionsData,
+});
+export const LongTitle = () => ({
+  components: { Task },
+  template: taskTemplate,
+  props: {
+    task: {
+      default: {
+        ...taskData,
+        title: longTitle,
+      },
+    },
+  },
```

Comment:
@IvanSotelo this needs also to be updated to be a factory function to prevent adding back the issue with the jest/vue cli.
it should be:
```javascript
task: {
      default: () => ({
        ...taskData,
        title: longTitle
      })
    }
```